src/components/Auth/Login/LoginForm.js

```diff
@@ -0,0 +1,65 @@
+import React, {Component} from 'react';
+import {Button, Col, Icon, Input, Row} from 'react-materialize';
+import  styles from './css/login.scss'
+
+
+class LoginForm extends Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			email: '',
+			emailError: '',
+			password: '',
+			passwordError: '',
+		};
+
+		this.handleChange = this.handleChange.bind(this);
+		this.handleSubmit = this.handleSubmit.bind(this);
+	}
+
+  handleChange(e) {
+  	this.setState({[e.target.name]: e.target.value});
+  }
+
+  handleSubmit(e) {
+  	if (e) e.preventDefault();
+  		this.setState({
```

Comment:
reformat this accordingly since you have a one liner on line 25

Reply:
Done